### PR TITLE
Zhiwen database table for subreddit entity and repo

### PR DESCRIPTION
**Pull request by HBzhainan-wzw**
# Overview

In a sentence or two, provide an overview of how this pull request updates the main branch.  You may use a bullet list if that helps.

# Issues Addressed

If this PR closes or deals with issues, mention them and tag the issue number with the `#` syntax.

# Details

Where applicable, include "before" and "after" screenshots or animated GIFs.   The tool [LiceCAP](https://www.cockos.com/licecap/) or one of [these similar tools](https://www.nextofwindows.com/5-free-tools-to-screen-capture-to-gif-on-windows) 
can be used to capture screenshots/animated GIFs.

## Reminders (delete this section before merging or as code is merged)
- [ ] Explicitly link to this pull request each issue that it closes
- [ ] Assign one and ideally two reviewers to review this pull request. 
- [ ] Reviewers should not just approve with "Looks good!" Instead, reviewers should carefully go through the code and use GitHub's reviewer interface to ask questions, make comments, and suggest changes.

### CODE REVIEW

File: src/main/java/edu/ucsb/cs156/team02/entities/CollegiateSubreddit.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  // This establishes that many todos can belong to one user
+  // Only the user_id is stored in the table, and through it we
+  // can access the user's details
+  private String name;
+  private String location;
+  private String subreddit;
+  private User user;
+  private String title;
+  private String details;
+  private boolean done;
```
Comment (andrewpengucsb) [inline src/main/java/edu/ucsb/cs156/team02/entities/CollegiateSubreddit.java]:
```suggestion
```
I believe these variables are unnecessary, and only needed in Todo.java and not CollegiateSubreddit.java

**Review by andrewpengucsb — CHANGES_REQUESTED** [review]

File: src/main/java/edu/ucsb/cs156/team02/entities/CollegiateSubreddit.java (hunk 1 above)
Comment (AlexLaiUCSB) [reply src/main/java/edu/ucsb/cs156/team02/entities/CollegiateSubreddit.java]: Deleted those variables in recent commit

**Review by andrewpengucsb — APPROVED** [review]

**Review by andrewpengucsb — APPROVED** [review]
LGTM

**Comment by AlexLaiUCSB** [comment]
- Fixed some typos in recent commit and tests compile and run correctly. 
- Storybook still not working, but should work after the other pull request goes through?

## Some screenshots of seeing database in Heroku 
image
image

- Think it can be merged.

**Pull request merged**